CHANGELOG.rst

```diff
@@ -23,6 +23,12 @@ Fixed
 Security
 ========
 
+[2.2.1] - 2019-03-12
+********************
+Fixed
+=====
+- Improvements code organization and fix some linter issues.
```

Comment:
Please, move changelogs to the UNRELEASED section. We can only add a release date when we make a real version release.

Also s/Improvements/Improve/

Reply:
Thanks for review. I updated the changelog.

Reply:
Thanks for review. Solved in last push.